docs/pages/2020_Intro_Week/notebooks/bandstructure.ipynb

```diff
@@ -0,0 +1,190 @@
+    "structure_Al = importer.query(id='9008460')[0].get_aiida_structure()\n",
+    "\n",
+    "structure_Al.get_formula()\n",
+    "\n",
+    "# The following structure can be used instead of Al, but will take much longer on the AWS machine.\n",
+    "# CaF2 COD ID='1000043' -- approximately 1/2 hour to run\n",
+    "# h-BN COD ID='9008997' -- approximately 45 mins to run\n",
+    "# GaAs COD ID='9008845' -- approximately 2 hours to run "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Now we run the bandstructure workchain for the selected structures"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The bandstructure workchain follows the following protocol:\n",
+    "* Determine the primitive cell of the input structure\n",
+    "* Run a vc-relax to relax the structure\n",
+    "* Refine the symmetry of the relaxed structure to ensure the primitive cell is used and run a self-consistent field calculation on it\n",
+    "* Run a non self-consistent field band structure calculation along a path of high symmetry k-points determined by [seekpath](http://materialscloud.org/tools/seekpath)\n",
+    "\n",
+    "Numerical parameters for the default 'theos-ht-1.0' protocol are determined as follows: \n",
+    "* Suitable pseudopotentials and energy cutoffs are automatically searched from the [SSSP library](http://materialscloud.org/sssp) installed on your machine  (it uses the efficiency version 1.1).\n",
+    "* K-point mesh is selected to have a minimum k-point density of 0.2 Å<sup>-1</sup>\n",
+    "* A Marzari-Vanderbilt smearing of 0.02 Ry is used for the electronic occupations\n",
+    "\n",
+    "In case the pseudopotentials are not installed, they can be downloaded in a terminal as:\n",
+    "\n",
+    "    mkdir sssp_pseudos\n",
+    "    wget 'https://archive.materialscloud.org/record/file?filename=SSSP_1.1_PBE_efficiency.tar.gz&record_id=23&file_id=d2ce4186-bf76-4e05-8b39-444b4da30273' -O SSSP_1.1_PBE_efficiency.tar.gz\n",
+    "    tar -C sssp_pseudos -zxvf SSSP_1.1_PBE_efficiency.tar.gz\n",
+    "    verdi data upf uploadfamily sssp_pseudos 'SSSP' 'SSSP pseudopotential library'\n",
+    "    \n",
+    "The protocol looks for a UPF file with a specific hash code, that is unique for each different file. \n",
+    "You can check that you have the right\n",
+    "one by performing a search in the database:\n",
+    "\n",
+    "    UpfData = DataFactory('upf')\n",
+    "    qb=QueryBuilder()\n",
+    "    qb.append(UpfData, filters={'attributes.md5':{'==':'cfc449ca30b5f3223ec38ddd88ac046d'}})\n",
+    "    len(qb.all())\n",
+    "\n",
+    "'md5' is a searchable attribute of the pseudopotential data object."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This will take approximately 6 minutes on the tutorial AWS (for Al)\n",
+    "results = run(\n",
```

Comment:
FYI, running this I get a deprecation warning:

```
/home/max/.virtualenvs/aiida/lib/python3.6/site-packages/plumpy/ports.py:194: UserWarning: the validator `validate_protocol` has a signature that only takes a single argument.
    This has been deprecated and the new signature is `validator(value, port)` where the `port` argument will be the
    port instance to which the validator has been assigned.
  warnings.warn(VALIDATOR_SIGNATURE_DEPRECATION_WARNING.format(self.validator.__name__))
07/08/2020 01:09:02 PM <6401> aiida.orm.nodes.process.workflow.workchain.WorkChainNode: [REPORT] [2167|PwBandStructureWorkChain|setup_protocol]: running the workchain with the "theos-ht-1.0" protocol
07/08/2020 01:09:04 PM <6401> aiida.orm.nodes.process.workflow.workchain.WorkChainNode: [REPORT] [2167|PwBandStructureWorkChain|run_bands]: launching PwBandsWorkChain<2182>
07/08/2020 01:09:05 PM <6401> aiida.orm.nodes.process.workflow.workchain.WorkChainNode: [REPORT] [2182|PwBandsWorkChain|run_relax]: launching PwRelaxWorkChain<2184>
07/08/2020 01:09:05 PM <6401> aiida.orm.nodes.process.workflow.workchain.WorkChainNode: [REPORT] [2184|PwRelaxWorkChain|run_relax]: launching PwBaseWorkChain<2187>
07/08/2020 01:09:06 PM <6401> aiida.orm.nodes.process.workflow.workchain.WorkChainNode: [REPORT] [2187|PwBaseWorkChain|run_calculation]: launching PwCalculation<2193> iteration #1
```

Reply:
but it does complete 👍

Reply:
yeah, that is just a warning from `plumpy` that the validator signature has been changed. The old workchain has not been adapted yet since it isn't really supposed to be used but just for demonstration purposes

Reply:
Indeed, I saw the warnings. For now I'll ignore them, we'll fix them in the next version of the tutorial. See #265